admin/lib/validator/check-window-v2/check-window-add-validator.js

```diff
@@ -204,68 +204,88 @@ checkWindowAddValidator.validate = (checkWindowData, validationConfig = null) =>
   // Compare date fields
   // Admin start date
   if (adminStartDate && adminStartDate.isAfter(adminEndDate, 'days')) {
+    // @ts-ignore
```

Comment:
Why are we applying `ts-ignore` to our own code?

Reply:
some of the code would require a large refactor to satisfy the typescript compiler.  The Ramda uses are typical of this.
We can certainly consider rewriting them, but i feel its outside the scope of this ticket.
at the very least we now have placeholders for code to address.
I would definitely be interested in having a deep dive with you on some of these, so we can understand the effort and direction we want to take

Reply:
Sounds like a plan